### PR TITLE
ci: run full spec test suite

### PR DESCRIPTION
**Pull request by gumb0**
Depends on #484

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -140,21 +140,21 @@ commands:
...
+        default: 681
 
     steps:
       - run:
           name: "Download spectest files"
           working_directory: ~/build
           command: |
             if [ ! -d wasm-spec ]; then
-              git clone https://github.com/wasmx/wasm-spec --branch vanilla-json --depth 1
+              git clone https://github.com/wasmx/wasm-spec --branch vanilla-json-full --depth 1
```
Comment (chfast) [inline circle.yml:157]: Why I'm not seeing this branch in wasmx/wasm-spec?

Can't we just use the same branch?

Comment (gumb0) [reply circle.yml:157]: It's there https://github.com/wasmx/wasm-spec/tree/vanilla-json-full

Maybe we can use the same one, if we merge test expectations update immediately after I push to it.

Comment (chfast) [reply circle.yml:157]: Ok, please do it now.

Comment (gumb0) [reply circle.yml]: Pushed to `vanilla-json`

**Pull request merged**